docs/changelog.md

```diff
@@ -4,6 +4,10 @@ Change Log
 ## Unreleased
 
 * Fix: Enum classes that only have an init block now also generate the required semicolon (#1952)
+* Fix: Preserve typealiases in `KSAnnotation.toAnnotationSpec()`. (#1945)
```

Comment:
nit:

```suggestion
* Fix: Preserve typealiases in `KSAnnotation.toAnnotationSpec()`. (#1956)
```

Reply:
This one actually linked the issue, so we link issues or fixes?

Reply:
I link fixes when compiling the changelog for a release, since fixes contain actually diffs. Does it make more sense to link issues?

Reply:
Up to you! I always just pick whichever one was contributed externally for their visibility as a default, but no strong feelings

Reply:
Gonna roll with this